tests/test_signature.py

```diff
@@ -9,6 +9,45 @@
 from sourmash import MinHash
 
 
+def test_minhash_copy(track_abundance):
+    e = MinHash(n=1, ksize=20, track_abundance=track_abundance)
+    e.add_kmer("AT" * 10)
+    sig = SourmashSignature(e, name='foo')
+    f = e.copy()
+    assert e == f
+
+
+def test_sig_copy(track_abundance):
+    e = MinHash(n=1, ksize=20, track_abundance=track_abundance)
+    e.add_kmer("AT" * 10)
+    sig1 = SourmashSignature(e, name='foo')
+    sig2 = sig1.copy()
+    assert sig1 == sig2
```

Comment:
ok - here, as with `MinHash` above, would be good to add a hash to `sig1.minhash` and verify that sig1 != sig2.

you could also add another test that changes the name - I think you can assign directly to `sig.name` - and verify that the copy's name doesn't change.

Reply:
So if I try to add a hash to `sig1` or `sig2` here, it gives me the same `TypeError` as below: `FrozenMinHash does not support modification`. 
I'm not sure I understand what you mean by the `sig.name` test. Could you please clarify?

Reply:
What should I do here?

Reply:
ok, now that I see that `SourmashSignature.__copy__` freezes the MinHash in the copy, I think this test is sufficient - thanks!